packages/lib/utils/logger.ts

```diff
@@ -0,0 +1,29 @@
+import pino from "pino";
+
+export const logger = pino({
+  level: "info",
+  timestamp: pino.stdTimeFunctions.isoTime,
+  transport: {
+    targets: [
+      {
+        level: "info",
+        target: "pino-pretty",
+        options: {
+          colorize: true,
+          levelFirst: true,
+          translateTime: "yyyy-dd-mm, h:MM:ss TT",
+        },
+      },
+      {
+        level: "info",
+        target: "pino/file",
+        options: {
+          destination: "../../formbricks.log",
```

Comment:
Could this have side effects depending on the deployment environment? Can we maybe first try without this to try the logger in production on Vercel and also don't fill the self-hosting containers with logs.

Reply:
Done! Please take a look now